AxonDeepSeg/network_construction.py

```diff
@@ -131,107 +94,87 @@ def uconv_net(x, training_config, phase, bn_updated_decay = None, verbose = True
         bn_decay = bn_updated_decay
 
     # Input picture shape is [batch_size, height, width, number_channels_in] (number_channels_in = 1 for the input layer)
-    net = tf.reshape(x, shape=[-1, image_size, image_size, 1])
-    data_temp = x
+
     data_temp_size = [image_size]
     relu_results = []
 
     ####################################################################
     ######################### CONTRACTION PHASE ########################
     ####################################################################
-    
+
+    # print(data_temp)
```

Comment:
Delete all the commented-out debugging print calls in this function

Reply:
Fixed in (328dcfc)